content/post/kubevault-v2022.12.28/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introducing KubeVault v2022.12.28
+date: "2022-12-28"
+weight: 25
+authors:
+- Abdullah Al Shaad
+tags:
+- cli
+- hashicorp
+- kubernetes
+- kubevault
+- kubevault-cli
+- redis
+- secret-management
+- security
+- vault
+---
+
+We are very excited to announce the release of KubeVault v2022.06.16 Edition. In the release KubeVault `v2022.12.28` a new `SecretEngine` for `RedisDB` has been added, `KubeVault CLI` has been updated accordingly. We're going to discuss some of them in details below.
```

Comment:
can we change it to something like: `KubeVault CLI` has been updated for generating `SecretProviderClass` for `Redis`?